### PR TITLE
ci: use dotnet test for uitests

### PR DESCRIPTION
**Pull request by kazo0**

### CODE REVIEW

File: samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Mobile/Android/AndroidManifest.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="Uno.Toolkit.Samples" android:versionCode="1" android:versionName="1.0">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="uno.platform.toolkit-uwp" android:versionCode="1" android:versionName="1.0">
```
Comment (jeromelaban) [inline samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Mobile/Android/AndroidManifest.xml:2]: I don't think it's the uwp appID that should be used here:

```suggestion
<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="uno.platform.toolkit" android:versionCode="1" android:versionName="1.0">
```

File: samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Mobile/Uno.Toolkit.Samples.Mobile.csproj [hunk 2]
```diff
@@ -5,6 +5,7 @@
     <!-- <TargetFrameworks>$(TargetFrameworks);net7.0-macos</TargetFrameworks> -->
     <SingleProject>true</SingleProject>
     <OutputType>Exe</OutputType>
+    <ApplicationId>uno.platform.toolkit-uwp</ApplicationId>
```
Comment (jeromelaban) [inline samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Mobile/Uno.Toolkit.Samples.Mobile.csproj:8]: Same here?

File: samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Mobile/iOS/Info.plist [hunk 3]
```diff
@@ -5,7 +5,7 @@
 	<key>CFBundleDisplayName</key>
 	<string>Uno.Toolkit.Samples</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.companyname.Uno.Toolkit.Samples</string>
+	<string>uno.platform.toolkit-uwp</string>
```
Comment (jeromelaban) [inline samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Mobile/iOS/Info.plist:8]: Same here

**Review by jeromelaban — APPROVED** [review]

File: samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Mobile/Uno.Toolkit.Samples.Mobile.csproj (hunk 2 above)
Comment (kazo0) [reply samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Mobile/Uno.Toolkit.Samples.Mobile.csproj:8]: Yeah this is the non-WinUI sample app, right? So we designated it with the -uwp suffix in the app id

Comment (pictos) [reply samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Mobile/Uno.Toolkit.Samples.Mobile.csproj:8]: @jeromelaban should remove the uwp suffix or not?

Comment (jeromelaban) [reply samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Mobile/Uno.Toolkit.Samples.Mobile.csproj:8]: Indeed, I missed that it's the uwp variant.

**Pull request merged**